src/nomad_material_processing/__init__.py

```diff
@@ -209,6 +505,32 @@ class SubstrateCrystalProperties(CrystalProperties):
         Section describing any miscut of the substrate with respect to the substrate
         orientation.
         """,
+        repeats=True,
+    )
+
+
+class ElectricProperties(ArchiveSection):
+    """
+    The electrical properties of a material.
+    """
+
+    m_def = Section()
+
+    conductivity_type = Quantity(
```

Comment:
Not sure about this name. This property is actually missing from the NOMAD properties taxonomy. Conductivity type sounds more like whether it is thermal, electric, or ionic. I know we're in the ElectronicProperties section, but still. Was this what Oliver called it? I would talk about the majority carrier type but that would map to holes or electrons... Maybe semiconductor_type? I think we would need to discuss this with a PI.